app/uk/gov/hmrc/individualsifapistub/domain/Income.scala

```diff
@@ -16,6 +16,142 @@
+import play.api.libs.json.Reads._
 
-case class CreateIncomeRequest(body: String)
+case class Address(line1: Option[String], line2: Option[String], line3: Option[String], line4: Option[String], postcode: Option[String])
+
+case class PostGradLoan(repaymentsInPayPeriod: Option[Double], repaymentsYtd: Option[Double])
+
+case class StudentLoan(
+                        planType: Option[String],
+                        repaymentsInPayPeriod: Option[Double],
+                        repaymentsYTD: Option[Double]
+                      )
+
+case class Benefits(taxedViaPayroll: Option[Double], taxedViaPayrollYtd: Option[Double])
+
+case class EmployeePensionContribs(
+                                    paidYtd: Option[Double],
+                                    notPaidYtd: Option[Double],
+                                    paid: Option[Double],
+                                    notPaid: Option[Double]
+                                  )
+
+case class GrossEarningsForNics(
+                                 inPayPeriod1: Option[Double],
+                                 inPayPeriod2: Option[Double],
+                                 inPayPeriod3: Option[Double],
+                                 inPayPeriod4: Option[Double]
+                               )
+
+case class EmployeeNics(
+                         inPayPeriod1: Option[Double],
+                         inPayPeriod2: Option[Double],
+                         inPayPeriod3: Option[Double],
+                         inPayPeriod4: Option[Double],
+                         ytd1: Option[Double],
+                         ytd2: Option[Double],
+                         ytd3: Option[Double],
+                         ytd4: Option[Double]
+                       )
+
+case class PayeEntry(
+                      taxCode: Option[String],
+                      paidHoursWorked: Option[String],
+                      taxablePayToDate: Option[Double],
+                      totalTaxToDate: Option[Double],
+                      taxDeductedOrRefunded: Option[Double],
+                      employerPayeRef: Option[String],
+                      paymentDate: Option[String],
+                      taxablePay: Option[Double],
+                      taxYear: Option[String],
+                      monthlyPeriodNumber: Option[String],
+                      weeklyPeriodNumber: Option[String],
+                      payFrequency: Option[String],
+                      dednsFromNetPay: Option[Double],
+                      employeeNics: Option[EmployeeNics],
+                      employeePensionContribs: Option[EmployeePensionContribs],
+                      benefits: Option[Benefits],
+                      parentalBereavement: Option[Double], // REMINDER TO SEL, THIS IS NESTED statutoryPayYTD / parentalBereavement
+                      studentLoan: Option[StudentLoan],
+                      postGradLoan: Option[PostGradLoan]
+                    )
+
+object PayeResponseObject {
```

Comment:
The ordering of objects is confusing here - perhaps place both objects at the bottom?